go.mod

```diff
@@ -135,3 +135,7 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.1 // indirect
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
+
+replace golang.org/x/crypto => golang.org/x/crypto v0.0.0-20220314234659-1baeb1ce4c0b
```

Comment:
The problem with such statements, in general, is that they end up sticking the version to a specific version which inevitably causes more exposure to CVEs. We need to be mindful that we are pinning the versions to a specific one and remove them once no longer needed.

Reply:
fully agree , but for the moment this is the only solution to respect what security's team is asking us